r/R/dplyr-funcs-datetime.R

```diff
@@ -463,3 +463,56 @@ duration_from_chunks <- function(chunks) {
   }
   duration
 }
+
+binding_as_date <- function(x,
+                            format = NULL,
+                            tryFormats = "%Y-%m-%d",
+                            origin = "1970-01-01") {
```

Comment:
I don't think we should change this here now, but something to think about and maybe do in the future if we have similar setups like this: It would probably be easier to pass the args down using `...` that way we don't always have to write out all of the arguments if we were to add one. 

Like I said, let's not change this now, but wanted to suggest that.